src/noos_inv/docker.py

```diff
@@ -1,31 +1,33 @@
 import os
 from typing import Optional
 
-from invoke import Collection, Context, task
+from invoke import Collection, Config, Context, task
 
 from . import utils
 
 
-CONFIG = {
-    "docker": {
-        # Sensitive
-        "repo": None,
-        "user": "AWS",
-        "token": None,
-        "arg": None,
-        # Non-sensitive
-        "file": "Dockerfile",
-        "context": ".",
-        "name": "webserver",
-        "tag": "test",
+CONFIG = Config(
```

Comment:
❓ 
Where is defined such a new API for collections in `invoke`'s documentation?

As per Invoke's configuration hierarchy:
https://docs.pyinvoke.org/en/stable/concepts/configuration.html#collection-configuration

Reply:
It was necessary for tests context: https://docs.pyinvoke.org/en/stable/api/context.html

I made an update to only change the context init in tests